Detectors/TPC/simulation/src/CommonModeContainer.cxx

```diff
@@ -0,0 +1,78 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file CommonModeContainer.cxx
+/// \brief Implementation of the Common Mode computation
+/// \author Andi Mathis, TU München, andreas.mathis@ph.tum.de
+
+#include "TPCSimulation/CommonModeContainer.h"
+
+#include "FairLogger.h"
+
+using namespace o2::TPC;
+
+void CommonModeContainer::addDigit(int cru, int timeBin, float signal)
+{
+  mEffectiveTimeBin = timeBin-mFirstTimeBin;
+  if(mEffectiveTimeBin < 0.) {
+    LOG(FATAL) << "TPC CommonMode buffer misaligned ";
```

Comment:
I just checked it again - the full message is shown, so it seems to be the FairLogger::endl that triggers the core dump